docs/release-notes/releaseposts/2024121101.md

```diff
@@ -0,0 +1,15 @@
+---
+draft: false
+date: 2024-12-11
+---
+
+# Media Connectors - OAuth JWT Bearer authentication
+
+![rn_icon](/assets/icon-GraFx-Studio.svg)
+
+<!-- more -->
+
+## Add support OAuth JWT Bearer authentication flow
+
+With this update, we added support to build connectors using an OAuth JWT Bearer authentication flow.
+See [documentation](/GraFx-Developers/connectors/authorization-for-connectors/?h=jwt#authorization-type-at) for more info.
```

Comment:
The link takes you to a page where "JWT" is highlighted. Is that intentional? Maybe you could use a link to the section header instead?